test/ninja/shell_command.py

```diff
@@ -53,6 +53,12 @@
 DefaultEnvironment(tools=[])
 
 env = Environment()
+
+# Added to verify that SCons Ninja tool is sanitizing the shell environment
+# before it spawns a new shell
+env['ENV']['ZPATH']=['/a/b/c','/c/d/e']
```

Comment:
is this supposed to be `ZPATH` or `PATH`?

Reply:
Yes ZPATH. It tests the issue but doesn't break PATH or other execution environment variable which is needed for the test to function.